eth-custodian/test/EthCustodianProxy.js

```diff
@@ -0,0 +1,266 @@
+const { ethers } = require('hardhat');
+const { expect } = require('chai');
+const { serialize } = require('rainbow-bridge-lib/rainbow/borsh.js');
+const { borshifyOutcomeProof } = require('rainbow-bridge-lib/rainbow/borshify-proof.js');
+
+const UNPAUSED_ALL = 0;
+const PAUSED_DEPOSIT_TO_EVM = 1 << 0;
+const PAUSED_DEPOSIT_TO_NEAR = 1 << 1;
+const PAUSED_WITHDRAW = 1 << 2;
+const PAUSED_WITHDRAW_PRE_MIGRATION = 1 << 3;
+const PAUSED_ALL = PAUSED_DEPOSIT_TO_EVM | PAUSED_DEPOSIT_TO_NEAR | PAUSED_WITHDRAW | PAUSED_WITHDRAW_PRE_MIGRATION;
+
+const SCHEMA = {
+  'Withdrawn': {
```

Comment:
Withdrawn -> BurnResult?

Reply:
The event is called `Withdrawn` https://github.com/aurora-is-near/eth-connector/blob/master/eth-custodian/contracts/EthCustodian.sol#L126

Reply:
No. 

In the link the `Withdrawn` event has two fields: recipient and amount. In this SHEMA the event has 3 fields: recipient, amount and ethCustodian.

It is different events. And I would like to rename it for not be confusing.

Reply:
Renamed to `WithdrawResult`

Reply:
Don't see the changes in this file